apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/service/NamespaceBoAsyncServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.ctrip.framework.apollo.portal.service;
+
+import com.ctrip.framework.apollo.common.dto.ItemDTO;
+import com.ctrip.framework.apollo.common.dto.ReleaseDTO;
+import com.ctrip.framework.apollo.portal.AbstractUnitTest;
+import com.ctrip.framework.apollo.portal.environment.Env;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author renjiahua
+ * @date 2021/1/17
+ */
+public class NamespaceBoAsyncServiceTest extends AbstractUnitTest {
+
+    @Mock
+    private ReleaseService releaseService;
+    @Mock
+    private ItemService itemService;
+    @InjectMocks
+    private NamespaceBoAsyncService namespaceBoAsyncService;
+
+    private String testAppId = "6666";
+    private String testClusterName = "default";
+    private String testNamespaceName = "application";
+    private Env testEnv = Env.DEV;
+
+    @Before
+    public void setup() {
+    }
+
+    @Test
+    public void testGetLatestReleaseAsync() {
```

Comment:
It doesn't smell right to catch the exception, in unit test we need to control the exceptions.

```suggestion
    public void testGetLatestReleaseAsync() throws ExecutionException, InterruptedException {
```